service-api/module/Application/src/Model/Entity/IdMethodIncludingNation.php

```diff
@@ -32,6 +32,9 @@ class IdMethodIncludingNation extends Entity
     #[Annotation\Validator(NotEmpty::class, options: [NotEmpty::NULL])]
     public string $id_country;
 
+    #[Annotation\Required(false)]
+    #[Annotation\Validator(NotEmpty::class, options: [NotEmpty::NULL])]
+    public ?string $id_correlation;
```

Comment:
This is specific to DWP, so I feel the name should reflect that. It could even go inside its own subobject on CaseData (like CounterService and IdentityIQ)